apis/v1alpha2/udproute_types.go

```diff
@@ -92,12 +92,17 @@ type UDPRouteRule struct {
 	// +kubebuilder:validation:MaxItems=8
 	Matches []UDPRouteMatch `json:"matches,omitempty"`
 
-	// ForwardTo defines the backend(s) where matching requests should
-	// be sent.
+	// BackendRefs defines the backend(s) where matching requests should be
+	// sent. If unspecified, this rule MUST not match.
 	//
-	// +kubebuilder:validation:MinItems=1
+	// Support: Core for Kubernetes Service
+	// Support: Custom for any other resource
+	//
+	// Support for weight: Extended
```

Comment:
I think we can communicate the "extended" weight support on that field based on the updated guidelines: https://github.com/kubernetes-sigs/gateway-api/pull/723/files#diff-5733e6fd292e68a0c67232b5291bcc3275622ef579fe6dc7893ffb202c7e217d.

Reply:
Can you elaborate a bit here? I'm not sure what is it that you are recommending to change and why.

Reply:
Sorry that wasn't very clear. That linked guidance basically states that a support level of a given field is the max of that field and any parent support levels. That would allow you to annotate the weight field directly with `Support: Extended`.

Reply:
> That would allow you to annotate the weight field directly with Support: Extended.

You mean Core? Weight field is set at Core for HTTPRoute.

Reply:
Oh woops, missed the context here, that is a tricky overlap. This seems fine then.